bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/BahmniVisitService.java

```diff
@@ -11,4 +12,6 @@ public interface BahmniVisitService {
     Visit getVisitSummary(String visitUuid);
 
     List<Encounter> getAdmitAndDischargeEncounters(Integer visitId);
+
+    Location getParentLocationForVisit(Location location);
```

Comment:
No need to introduce this on the interface. This can very well be in SharePrescriptionService implmentation

Reply:
moved to SharePrescriptionServiceImpl